serverless/functions/how-to/test-a-function.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to test a function
+  description: This page explains how to test a function
+content:
+  h1: How to test a function
+  paragraph: This page explains how to test a function
+tags: function serverless test-function
+dates:
+  validation: 2023-09-25
+  posted: 2023-09-25
+categories:
+  - serverless
+---
+
+This page shows you how to test a function.
```

Comment:
Maybe add a little more info on what is means to test a function or what information it allows you to get? (so it's a little bit different from the title of the page)